components/single-form.jsx

```diff
@@ -136,13 +135,8 @@ module.exports = React.createClass({
     this.stripeCheckout(validate, submit, this.props.billingAddress);
   },
   renderPrivacyPolicy: function() {
-    if (this.props.privacyPolicyTest) {
```

Comment:
You need to remove this attribute, and the code that changes the list of required options based on this code too